service/resource/legacyv2/adapter/vpc.go

```diff
@@ -8,6 +8,28 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+func VPCID(clients Clients, name string) (string, error) {
```

Comment:
Here we get the VPC ID. Once the VPC is created by Cloud Formation we can remove this and use a reference in the template.